presto-docs/src/main/sphinx/release/release-0.244.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.244
+=============
+
+**Highlights**
+==============
+* Improve performance of NestedLoopJoinOperator.
+* Add support for spilling joins when using materialized exchanges.
+
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an issue where queries could encounter an int overflow error during spilling.
+* Fix :func:`hamming_distance` to handle code point zero correctly.
+* Fix caching and case-sensitivity bugs with type signatures for enum types.
+* Fix LambdaDefinitionExpression canonicalization to correctly canonicalize Block constant (:issue:15424).
+* Improve performance of NestedLoopJoinOperator.
+* Add `peak_node_total_memory` to statistics to allow monitoring the peak sum of memory used across tasks at any node by a query.
+* Add support for spilling joins when using materialized exchanges.
+* Add thrift transport support for TaskStatus which can be enabled using config property ``experimental.internal-communication.thrift-transport-enabled``.
+
+Druid Changes
+_____________
+* Fix druid reading issue caused by using the ARRAY_LINES result format.
+
+Hive Changes
+____________
+* Fix a bug with reading encrypted DWRF tables where queries could fail with a NullPointerException.
+* Add support for AWS SDK client request metrics and Glue API metrics to GlueHiveMetastore.
+
+Pinot Changes
+_____________
+* Add support for querying Pinot server with streaming API (with Pinot version >= 0.6.0); The API can be enabled by setting ``pinot.use-streaming-for-segment-queries=true``.
+
+
+Verifier Changes
+________________
+* Fix an issue where queries are always directed to the first cluster when multiple control or test clusters are specified.
+* Improve verifier handling of memory limits by using the limits from the verifier cluster which are frequently better matched to the queries picked to be run in the verifier suite.
```

Comment:
This line is too long and complicated.

```
Improve handling of memory limits by using the verifier cluster's memory limits instead of the limits defined while running the query.
```